site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -4,6 +4,26 @@
 [field]#sourceTenantId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.14.0#::
 The optional Id of an existing Tenant to make a copy of. If present, the [field]#tenant.id# and [field]#tenant.name# values of the request body will be applied to the new Tenant, all other values will be copied from the source Tenant to the new Tenant.
 
+[field]#tenant.captchaConfiguration.captchaMethod# [type]#[String]# [required]#Required# [since]#Available since 1.30.0#::
```

Comment:
Need to update all of these with the `shared/_premium-edition-blurb-api.adoc` fragment. See `tenant.formConfiguration.adminUserFormId` for an example.

That only applies to the request objects.